ports/libid3tag/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_sourceforge(
```

```suggestion
vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")

vcpkg_from_sourceforge(
```
Seems this port doesn't support arm and uwp, so it would be better to add the unsupported message.

---

What is the difference between vcpkg_fail_port_install and `supports` in vcpkg.json?

---

` vcpkg_fail_port_install` is mainly used to remind users. When installing the port, it will post the error info if the triplets are not supported.

`supports `filed is focus on CI testing. If we add this field, CI testing will skip the test on the related triplets. 

https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_fail_port_install.md
https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#supports

---

That's confusing and error-prone to duplicate the information in two places.

---

I added the suggested line but it's more restrictive than what I can express in vcpkg.json. ARM is only not supported on Windows, but that doesn't mean this won't build on ARM on Linux or macOS.

---

I used `message(FATAL_ERROR)` to handle that instead of the `vcpkg_fail_port_install` macro.